odata-url-conventions/5 Query Options.md

```diff
@@ -2316,6 +2316,7 @@ Search expressions are used within the
 entities matching the specified expression.
```

Comment:
```suggestion
entities matching the specified expression. Leading and trailing spaces are not considered part of the search expression.
```
Covers cases like `$search=%20"%20"%20-%20` which searches for space AND hyphen.